chainerx_cc/chainerx/routines/math.cc

```diff
@@ -258,8 +258,50 @@ Array Multiply(Scalar x1, const Array& x2) { return Multiply(x2, x1); }
 
 namespace {
 
+void FloorDivideImpl(const Array& x1, const Array& x2, const Array& out) {
+    // TODO(imanishi): dtype conversion
+    CheckEqual(x1.dtype(), x2.dtype());
+    CheckEqual(x1.shape(), x2.shape());
+
+    {
+        NoBackpropModeScope scope{};
+        x1.device().FloorDivide(x1, x2, out);
+    }
+
+    {
+        BackwardBuilder bb{"floor_divide", {x1, x2}, out};
+        if (BackwardBuilder::Target bt = bb.CreateTarget(0)) {
+            bt.Define([](BackwardContext& bctx) {
+                const Array& gout = *bctx.output_grad();
+                bctx.input_grad() = gout * Scalar{0, gout.dtype()};
```

Comment:
Is there any better implementation?

Reply:
Does this operation need backward implementation?

Reply:
Removed.